monix-catnap/src/main/scala/com/avast/sst/monix/catnap/CircuitBreakerModule.scala

```diff
@@ -0,0 +1,14 @@
+package com.avast.sst.monix.catnap
+
+import cats.effect.{Clock, Sync}
+import monix.catnap.CircuitBreaker
+
+object CircuitBreakerModule {
+
+  /** Makes [[monix.catnap.CircuitBreaker]] initialized with the given config and optionally [[cats.effect.Clock]]. */
+  def make[F[_]: Sync](config: CircuitBreakerConfig, clock: Option[Clock[F]] = None): F[CircuitBreaker[F]] = {
+    implicit val finalClock: Clock[F] = clock.getOrElse(Clock.create[F])
+    CircuitBreaker.of[F](config.maxFailures, config.resetTimeout, config.exponentialBackoffFactor, config.maxResetTimeout)
```

Comment:
Tomas, you're mixing internal stuff with public OSS project :) There is no public circuit breaker implementation with `F` that I am aware of so this is our only option right now. I know this "module" does not provide much but the whole point is to provide "the same feel" to everything which means we need to have a configuratio case class and provide PureConfig readers so that it can be used together with the rest of the toolkit.

Reply:
> Tomas, you're mixing internal stuff with public OSS project :) 

i dont understand that what you mean by that ... but anyway

that's fine, but what i would expect / want is to have a full featured circuit breaker with monitoring ... what i'm saying is that i would hope THIS toolkit would provide that

Reply:
All possible. Just that it first needs to be open source and it shouldn't prevent us from using Monix CircuitBreaker, we can have both.

Reply:
👍